src/components/productCardShowMoreButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Image from "next/image";
+
+type ContentProps = {
+  backOfCardImage?: string[];
+  image: string[];
+};
+
+type ProductCardProps = {
+  content: ContentProps;
+  isCardFlipped: boolean;
+  changeImage?: (newImage?: string) => void;
+  toggleIsFlippedCard: () => void;
+};
+
+const backOfCardImageIsValid = (ContentProps: ContentProps) => {
+  if (ContentProps.backOfCardImage !== undefined) {
+    return ContentProps.backOfCardImage[0];
+  }
+};
+
+export default function productCardShowMoreButton({
+  content,
+  isCardFlipped,
+  changeImage,
+  toggleIsFlippedCard,
+}: ProductCardProps) {
+  const frontProductImage = content.image[0];
+  const backProductImage = backOfCardImageIsValid(content);
```

Comment:
Again, when I read this, backOfCardImageIsValid reads as if it's suppose to be a boolean

Reply:
how would you go about naming the function then? My previous naming implies a boolean; I would assume since I'm checking if something is valid and ∴ would return true or false. Would something like backOfImageIsImageIsDefined or imageIsAvaialbe would be more apprioate naming?

Reply:
I probably wouldn't use a helper function here unless I need to use it multiple places. Just to check to see if it's available, if it is, assign it, else no backProductImage

Reply:
understood.